src/app/users/view-user/view-user.component.html

```
@@ -61,11 +61,13 @@
         <div class="content-row">
           <div class="label">{{ 'labels.buttons.Is Self Service' | translate }}</div>
           <div class="value">
-            <span
-              [ngClass]="userData.selfService ? 'self-service-true' : 'self-service-false'"
-              class="self-service-indicator"
-            ></span>
-            {{ userData.selfService ? ('labels.buttons.Yes' | translate) : ('labels.buttons.No' | translate) }}
+            <span style="display: inline-flex; align-items: center; gap: 0.5rem">
```

HTMLHint (1.8.0)

[warning] Inline style [  style="display: inline-flex; align-items: center; gap: 0.5rem" ] cannot be used.

(inline-style-disabled)